docs/source/configuration/authn-jwt.mdx

```diff
@@ -156,6 +158,8 @@ Below are 2 example [Rhai script](../customizations/rhai/) customizations that d
 
 ### Example: Forwarding claims to subgraphs
 
+> ⚠️ This example forwards JWT claims via headers. When combined with [header propagation](./header-propagation/) this can introduce a security risk. If you are relying on header-based forwarding, **be sure not to propagate matching headers**.
```

Comment:
```suggestion
> ⚠️ This example forwards JWT claims via headers. When combined with [header propagation](./header-propagation/) this can introduce a security risk. If you're relying on header-based forwarding, **make sure not to propagate matching headers**.
```